### PR TITLE
[FIX] Fixed some issues with driver pages

### LINKED ISSUE #10718: Drivers: Unable to deactivate drivers in bulk
**Setup**
- Rancher version: v2.9-head
- Rancher UI Extensions:
- Browser type & version: Any

**Describe the bug**
User is unable to `deactivate` cluster/node drivers in bulk. 

**To Reproduce**
1. login to Rancher as an admin
2. go to the Cluster Management page
3. click 'Drivers' tab in the side nav
4. click Cluster/Node Drivers tab
5. select two or more active drivers
6. click deactivate
7. notice only one of the drivers selected were deactivated

**Result**
User is unable to `deactivate` cluster/node drivers in bulk. 

**Expected Result**
User is able to `deactivate` cluster/node drivers in bulk. 

**Screenshots**

https://github.com/rancher/dashboard/assets/127343932/2552b278-0256-4060-8f65-ab1938848e34

**Additional context**
We are currently skipping the associated e2e tests in `kontainer-drivers.spec.ts` and `node-drivers.spec.ts`. Please enable them once this issue has been resolved by removing `.skip`.

### LINKED ISSUE #11046: Cluster Management: Drivers: id is used instead of name in the cluster/node driver list view
**Setup**
- Rancher version: v2.9-head
- Rancher UI Extensions:
- Browser type & version: Any

**Describe the bug**
Upon creation of a cluster/node driver, `id` is used instead of `name` in the list view which is inconsistent with the previous behavior in 2.8 (before the redesign).
 

**To Reproduce**
1. login to Rancher as an admin
2. go to the Cluster Management page
3. click 'Drivers' tab on the side nav
4. click 'Cluster Drivers' tab
5. click 'Create'
6. paste in download URL (https://github.com/rancher-plugins/kontainer-engine-driver-example/releases/download/v0.2.3/kontainer-engine-driver-example-copy1-linux-amd64)
7. click Create
8. notice `id` is used instead of `name`

**Result**
`id` is used instead of `name` in the cluster/node driver list view. 

**Expected Result**
`name` is used in the cluster/node driver list view, not `id`.

**Screenshots**
current behavior (in 2.9):
Screenshot 2024-05-16 at 11 20 20 AM

previous behavior (in 2.8): 

https://github.com/rancher/dashboard/assets/127343932/fa92a271-cffd-4eac-a104-9069ef8a47c1

**Additional context**
Need to update existing e2e tests (`kontainer-drivers.spec.ts` and `node-drivers.spec.ts`) to be sure this is covered; just need to uncomment the assertions.

### PR DESCRIPTION
**Pull request by eva-vashkevich**
### Summary
Fixes #11046
Fixes #10718 

Fixed deactivate driver bulk action, fixed displaying driver id instead of name for drivers that do not have translation.

### Occurred changes and/or fixed issues

Additionally, changed bulk activation action to follow pattern similar to other models.
When provisioning a cluster, driver name is used instead of the id.

### Technical notes summary

N/A

### Areas or cases that should be tested

[] Test that driver name is shown instead of id for non-builtin driver
[] Test that driver can be activated on its own
[] Test that drivers can be activated in bulk
[] Test that driver can be deactivated on its own
[] Test that drivers can be deactivated in bulk 

### Areas which could experience regressions

- Driver name is displayed instead of id inside of the cluster creation wizard.

### Checklist
- [x] The PR is linked to an issue and the linked issue has a Milestone, or no issue is needed
- [x] The PR has a Milestone 
- [x] The PR template has been filled out
- [x] The PR has been self reviewed 
- [x] The PR has a reviewer assigned
- [x] The PR has automated tests or clear instructions for manual tests and the linked issue has appropriate QA labels, or tests are not needed
- [x] The PR has reviewed with UX and tested in light and dark mode, or there are no UX changes

### CODE REVIEW

**Comment by nwmac** [comment]
@aalves08 @richard-cox Adding you as reviewers, in case you have time to help review this and get this in.

File: shell/dialog/DeactivateDriverDialog.vue [hunk 1]
```diff
@@ -34,10 +51,14 @@ export default {
...
     async apply(buttonDone) {
       try {
-        await this.$store.dispatch('rancher/request', {
-          url:    this.url,
-          method: 'post'
-        });
+        await Promise.all(this.drivers.map((driver) => {
+          const url = `v3/${ this.driverType }/${ escape(driver.id) }?action=deactivate`;
+
+          this.$store.dispatch('rancher/request', {
```
Comment (richard-cox) [inline shell/dialog/DeactivateDriverDialog.vue:57]: this promise should be returned

**Review by richard-cox — CHANGES_REQUESTED** [review]
only one small comment, otherwise looking good

**Review by richard-cox — APPROVED** [review]

**Comment by richard-cox** [comment]
If there are no overnight objections to my tweak, will merge tomorrow AM UK time

**Review by richard-cox — APPROVED** [review]

**Comment by aalves08** [comment]
@richard-cox I think Eva is out, right? If it's approved can it be merged?

**Comment by richard-cox** [comment]
Yep, just been tickling gates after rebase.

**Pull request merged**